src/config/fact-core.toml.example

```diff
@@ -0,0 +1,97 @@
+[common.redis]
+fact-db = 3
+test-db = 13
+host = "localhost"
+port = 6379
+
+[common.logging]
+level = "WARNING"
+file = "/tmp/fact_main.log"
+
+[backend]
+firmware-file-storage-directory = "/media/data/fact_fw_data"
+temp-dir-path = "/tmp"
+docker-mount-base-dir = "/tmp/fact-docker-mount-base-dir"
+block-delay = 0.1
+ssdeep-ignore = 1
+communication-timeout = 60
+intercom-poll-delay = 1.0
+throw-exceptions = false
+
+
+[backend.authentication]
+enabled = false
+user-database = "sqlite:////media/data/fact_auth_data/fact_users.db"
+password-salt = "5up3r5tr0n6_p455w0rd_5417"
+
+
+
+[backend.postgres]
+server = "localhost"
+port = 5432
+database = "fact_db"
+test-database = "fact_test"
+ro-user = "fact_user_ro"
+ro-pw = "change_me_ro"
+rw-user = "fact_user_rw"
+rw-pw = "change_me_rw"
+del-user = "fact_user_del"
+del-pw = "change_me_del"
+admin-user = "fact_admin"
+admin-pw = "change_me_admin"
+
+[backend.unpacking]
+processes = 4
+whitelist = [
+    "application/x-object",
+    "application/x-shockwave-flash",
+    "audio/mpeg",
+    "image/gif",
+    "image/jpeg",
+    "image/png",
+    "text/plain",
+    "video/mp4",
+    "video/mpeg",
+    "video/ogg",
+    "video/quicktime",
+    "video/x-msvideo",
+]
+max-depth = 8
+memory-limit = 2048
+threshold = 0.8
+throttle-limit = 50
+
+
+[[backend.plugin]]
+name = "cpu_architecture"
+processes = 4
+
+[[backend.plugin]]
+name = "cve_loookup"
+processes = 2
+
+[[backend.presets]]
```

Comment:
`backend.preset` without an s is probably better than this. Also its more symmetric with `backend.plugin`.